app/script/components/groupVideoGrid.js

```diff
@@ -58,31 +58,59 @@ z.components.GroupVideoGrid = (() => {
       this.streams.subscribe(this.updateGrid.bind(this));
       this.updateGrid(this.streams());
 
+      this.scaleVideos = this.scaleVideos.bind(this, rootElement);
+      // scale videos when the grid is updated (on the next rendering cycle)
+      this.grid.subscribe(() => {
+        window.requestAnimationFrame(this.scaleVideos);
+      });
+
+      // scale the videos when the window is resized
+      window.addEventListener('resize', this.scaleVideos);
+
+      this.rootElement = rootElement;
```

Comment:
you already bound it to the function, do you still need it somewhere else?

Reply:
very good point. It was needed for the previous iteration and I forgot to remove it :D 
:+1: